pallets/ddc-customers/src/tests.rs

```diff
@@ -289,3 +319,91 @@ fn unlock_and_withdraw_deposit_works() {
 		assert_eq!(DdcCustomers::ledger(&account_1), None);
 	})
 }
+
+#[test]
+fn set_bucket_params_works() {
+	ExtBuilder.build_and_execute(|| {
+		System::set_block_number(1);
+
+		let cluster_id = ClusterId::from([1; 20]);
+		let bucket_owner = 1;
+		let bucket_params = BucketParams { is_public: false };
+
+		// Bucket created
+		assert_ok!(DdcCustomers::create_bucket(
+			RuntimeOrigin::signed(bucket_owner),
+			cluster_id,
+			bucket_params
+		));
+
+		// Checking that event was emitted
+		assert_eq!(System::events().len(), 1);
+		System::assert_last_event(Event::BucketCreated(1u64).into());
+
+		let bucket_id = 1;
+		let update_bucket_params = BucketParams { is_public: true };
```

Comment:
What happens if the bucket is private in this case? I don't see anything that uses this flag in this PR.

Reply:
There are no references to this field inside the protocol at the moment. Its usage is limited to the DDC storage node that reads this field when it decides whether to give access to the bucket content to external users or not